UnitsNet/CustomCode/Quantities/Length.extra.cs

```diff
@@ -236,5 +236,70 @@ public string ToString(IFormatProvider? cultureInfo)
             // So inches are rounded when converting from base units to feet/inches.
             return string.Format(cultureInfo, "{0:n0} {1} {2:n0} {3}", Feet, footUnit, Math.Round(Inches), inchUnit);
         }
+
+        /// <summary>
+        ///     Outputs feet and inches on the format: {feetValue}' - {inchesValueIntegral}[ {inchesValueFractional}]"
+        ///     The inches are rounded to the nearest fraction of the fractionDenominator argument and reduced over the greatest common divisor.
+        /// </summary>
+        /// <param name="fractionDenominator">The maximum precision to express the rounded inch fraction part</param>
+        /// <example>Length.FromFeetInches(3,2).ToArchitecturalString(32).ToString() outputs: "3 ft 2 in"</example>
```

Comment:
Shouldn't the example output string be `3' - 2"` ?

Reply:
I would love a few more examples to help illustrate how different `fractionDenominator` values determines the output for the same input.

For example:
```xmldoc
<example>
<code>
var length = Length.FromFeetInches(3, 2.6);
length.ToArchitecturalString(1)   => 3' - 3"
length.ToArchitecturalString(2)   => 3' - 2 1/2"
length.ToArchitecturalString(4)   => 3' - 2 1/2"
length.ToArchitecturalString(8)   => 3' - 2 5/8"
length.ToArchitecturalString(16)  => 3' - 2 5/8"
length.ToArchitecturalString(32)  => 3' - 2 19/32"
length.ToArchitecturalString(128) => 3' - 2 77/128"
</code>
</example>
```